src/generate/renders/generated/CliCommands.ts

```diff
@@ -75,6 +77,8 @@ export class CliCommands extends TemplateBase {
         } else if (this.showCustomFunctionName !== '') {
             this.showCustomFunctionName = '';
         }
+        this.groupClientFactoryName = item['clientFactoryName'];
+        this.importClientFactories.push(item['clientFactoryName']);
```

Comment:
push this.groupClientFactoryName directly to avoid accessing array again.

Reply:
changed.